R/FilterState.R

```diff
@@ -141,7 +139,7 @@ FilterState <- R6::R6Class( # nolint
     #' - `fixed` state is prevented from changing state
     #' - `anchored` state is prevented from removing state
     #'
-    #' @param state a `teal_slice` object
+    #' @param state (`teal_slice`) object
```

Comment:
@kartikeyakirar in `FilterState` where you have `@param slice`  we just write `(teal_slice)` without `object`. Can you unify that? I would opt for the removal of `object` in `@param` specifications

Reply:
done [95ced22](https://github.com/insightsengineering/teal.slice/pull/506/commits/95ced22df80e0414a42be5ca506cdcd43a3e9a92)